docs/tutorials/gluon/hybrid.md

```diff
@@ -137,4 +137,94 @@ to gluon with `SymbolBlock`:
 net2 = gluon.SymbolBlock.imports('model-symbol.json', ['data'], 'model-0001.params')
 ```
 
+## Operators that does not work with hybridize
+
+If you want to hybridize your model, you must use `F.some_operator` in your 'hybrid_forward' function, F will be 'mxnet.nd' before you hybridize and 'mxnet.sym' after hybridize. While most APIs are the same in NDArray and Symbol,  there are some differences. Writing `F.some_operator` and call `hybridize` may not work all the time.
+Here we list some frequently used NDArray APIs that can't be hybridized and provide you the work arounds.
+
+### Element-wise Operators
+
+The following arithmetic and comparison APIs are automatically broadcasted if the input NDArrays have different shapes.
+However, that's not the case in Symbol API, it's not automatically broadcasted and you have to manually specify whether to use element-wise operator or broadcast operators.
+
+
+| NDArray APIs  | Description  |
```

Comment:
This table is a little hard to read. Could the `<=>` be replaced with columns instead?